setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read().replace('.. :changelog:', '')
+
+requirements = [
+    # TODO: put package requirements here
+]
+
+test_requirements = [
+    # TODO: put package test requirements here
+]
+
+setup(
+    name='panamadv',
+    version='0.1.1',
+    description="Program to calculate the DV for the given RUC for Panama.",
+    long_description=readme + '\n\n' + history,
+    author="Jose Vasconcellos",
+    author_email='info@vauxoo.com',
```

Comment:
First off, thank you for addressing the license issue and I'm sorry if I've
overreacted.
For the email, please use vasmac+github@gmail.com.

I'll update the copyright issue this weekend.

On Thu, Sep 3, 2015 at 8:14 PM, Olar Alca [Vauxoo] <notifications@github.com
> wrote:

> In setup.py
> <https://github.com/apple314159/panama-dv/pull/4#discussion_r38714094>:
>
> > +
> > +requirements = [
> > +    # TODO: put package requirements here
> > +]
> > +
> > +test_requirements = [
> > +    # TODO: put package test requirements here
> > +]
> > +
> > +setup(
> > +    name='panamadv',
> > +    version='0.1.1',
> > +    description="Program to calculate the DV for the given RUC for Panama.",
> > +    long_description=readme + '\n\n' + history,
> > +    author="Jose Vasconcellos",
> > +    author_email='info@vauxoo.com',
>
> @apple314159 <https://github.com/apple314159> can you tell provide me a
> vali e-mail addres to put it in here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apple314159/panama-dv/pull/4/files#r38714094>.
>